src/htmx.js

```diff
@@ -1337,6 +1337,22 @@ return (function () {
                     }
                     consumeUntil(tokens, NOT_WHITESPACE);
                 } while (tokens[0] === "," && tokens.shift())
+                if (htmx.config.triggerSpecsCache) {
+                    htmx.config.triggerSpecsCache[explicitTrigger] = triggerSpecs
+                }
+                return triggerSpecs
+        }
+
+        /**
+         * @param {HTMLElement} elt
+         * @returns {import("./htmx").HtmxTriggerSpecification[]}
+         */
+        function getTriggerSpecs(elt) {
+            var explicitTrigger = getAttributeValue(elt, 'hx-trigger');
+            var triggerSpecs = [];
+            if (explicitTrigger) {
+                triggerSpecs = (htmx.config.triggerSpecsCache && htmx.config.triggerSpecsCache[explicitTrigger])
+                    || parseAndCacheTrigger(elt, explicitTrigger)
```

Comment:
Super nit, but would you mind assigning the cache to a local var, something like `cache`, and then passing it in to `parseAndCacheTrigger` as a third parameter, just to avoid so much `htmx.config.triggerSpecsCache` in the code?